attributes/default.rb

```diff
@@ -53,3 +53,4 @@
 ]
 default['spamassassin']['spamd']['pidfile'] = '/var/run/spamd.pid'
 default['spamassassin']['spamd']['nice'] = nil
+default['spamassassin']['conf']['localrules'] = []
```

Comment:
Could you add an example to the readme about how to use it?

Reply:
Will do.

Reply:
Done